tests/unit/spring_boot_patch.py

```diff
@@ -182,10 +235,16 @@ def __init__(self) -> None:
         """Initialize the :class:`SpringBootPatch` instance."""
         self.container_mocks: typing.Dict[str, ContainerMock] = {}
         self.images: typing.Dict[str, OCIImageMock] = {}
-        self._patch = patch.multiple(
-            ops.model.Unit,
-            get_container=self._gen_get_container_mock(ops.model.Unit.get_container),
+        self._patches: typing.List[_patch] = []
+        self._patches.append(
+            patch.multiple(
```

Comment:
Would it be worth adding the `spec` argument here?

Reply:
Added, thanks!